contrib/mpi-proxy-split/mpi-wrappers/mpi_unimplemented_wrappers.txt

```diff
@@ -155,7 +149,6 @@ int MPI_Igather(const void *sendbuf, int sendcount, MPI_Datatype sendtype, void
 int MPI_Igatherv(const void *sendbuf, int sendcount, MPI_Datatype sendtype, void *recvbuf, const int recvcounts[], const int displs[], MPI_Datatype recvtype, int root, MPI_Comm comm, MPI_Request *request);
 int MPI_Get_address(const void *location, MPI_Aint *address);
 int MPI_Get_elements(const MPI_Status *status, MPI_Datatype datatype, int *count);
```

Comment:
Assuming that this PR  will add an implementation of `MPI_Get_elements` (see suggestion above), we should remove it here from `mpi_unimplementeed_wrappers.txt`.